### PR TITLE
Show saved in progress forms on Burials landing pages

### PR DESCRIPTION
**Pull request by cehsu**
SIP cards have been added to existing Burials landing pages, including the main page as well as eligibility. There isn't an apply page, so one question is whether one should be created as part of this set of changes, or flagged for future work?

### Burials and Memorials (saved form)
localhost-3001-burials-and-memorials-

### Burial Allowances and Survivor Compensation (saved form)
localhost-3001-burials-and-memorials-survivor-and-dependent-benefits- 3

### Burial Allowance Application Process (unauthed/no saved form)
localhost-3001-burials-and-memorials-survivor-and-dependent-benefits-burial-costs-

### Burial Allowance Application Process (saved form)
localhost-3001-burials-and-memorials-survivor-and-dependent-benefits-burial-costs- 2

### HCA (unauthed/no saved form)
localhost-3001-health-care-eligibility- 1
localhost-3001-health-care-apply- 1

### HCA (saved form)
localhost-3001-health-care-eligibility-
localhost-3001-health-care-apply-

### Pension (unauthed/no saved form)
localhost-3001-pension-eligibility- 1
localhost-3001-pension-apply- 1

### Pension (saved form)
localhost-3001-pension-eligibility-
localhost-3001-pension-apply-

### CODE REVIEW

File: src/js/no-react-entry.js [hunk 1]
```diff
@@ -28,7 +29,7 @@ createLoginWidget(store);
 if (pensionPages.has(location.pathname)) {
   createApplicationStatus(store, {
     formId: '21P-527EZ',
-    applyText: 'Apply for Veterans Pension'
+    applyText: 'Apply for Veterans Pension Benefits'
```
Comment (cehsu) [inline src/js/no-react-entry.js]: Updated in response to this [feedback](https://github.com/department-of-veterans-affairs/vets-website/pull/6611/files#r145828134).

File: content/pages/burials-and-memorials/eligibility.md [hunk 2]
```diff
@@ -55,3 +60,13 @@ We give the surviving spouse, children, and parents of deceased Servicemembers a
...
 
 **Please note:** The pre-need eligibility determination is only for national cemeteries. If you want to be buried in a state or tribal Veterans cemetery, check the site you want ahead of time, since some state Veterans cemeteries require that you live in that state or have other rules around eligibility. [Find a state Veterans cemetery](http://www.cem.va.gov/cem/cems/listcem.asp).
+
+### Ready to apply?
+
+<div id="react-applicationStatus">
+  <a class="usa-button-primary va-button-primary" href="/burials-and-memorials/application/530/">Apply for Burial Benefits</a>
+</div>
+
+[Learn about the application process](/burials-and-memorials/application/530/).
```
Comment (cehsu) [inline content/pages/burials-and-memorials/eligibility.md]: Is this where this should route?

File: src/js/no-react-entry.js [hunk 3]
```diff
@@ -5,6 +5,7 @@ import createApplicationStatus from './common/components/createApplicationStatus
 const wizardPages = new Set(['/education/apply/', '/education/eligibility/']);
 const pensionPages = new Set(['/pension/', '/pension/apply/', '/pension/eligibility/']);
 const healthcarePages = new Set(['/health-care/', '/health-care/apply/', '/health-care/eligibility/']);
+const burialPages = new Set(['/burials-and-memorials/', 'burials-and-memorials/eligibility/']);
```
Comment (cehsu) [inline src/js/no-react-entry.js]: There is not currently an apply page, should one be created? Should that be part of this set of changes? Or should a ticket be written for this?

File: content/pages/burials-and-memorials/eligibility.md [hunk 4]
```diff
@@ -55,3 +60,13 @@ We give the surviving spouse, children, and parents of deceased Servicemembers a
 Yes. You can plan ahead to make the process of applying for a burial in a VA national cemetery easier for your family members in the event of your death. To do this, you’ll need to apply for a pre-need eligibility determination. [Find out how to apply](/burials-and-memorials/pre-need/).
 
 **Please note:** The pre-need eligibility determination is only for national cemeteries. If you want to be buried in a state or tribal Veterans cemetery, check the site you want ahead of time, since some state Veterans cemeteries require that you live in that state or have other rules around eligibility. [Find a state Veterans cemetery](http://www.cem.va.gov/cem/cems/listcem.asp).
+
+### Ready to apply?
+
```
Comment (cehsu) [inline content/pages/burials-and-memorials/eligibility.md]: This entire section has been added, was it not here for a specific reason before? Should it not be added now?

**Review by stacyhlinn — COMMENTED** [review]
Burial pages are structured differently so yeah it is different than the other benefits. 
We should also add a SiP here: https://staging.vets.gov/burials-and-memorials/survivor-and-dependent-benefits/

#5382 has a comp.

**Review by peggygannon — CHANGES_REQUESTED** [review]
@cehsu Hi, Claire, a few comments from me: 
Screen 1: looks good
Screen 2:  Please remove the Ready to Apply?  and apply button to the Burial Allowances and Survivor Compensation landing page. We need further discussion with the Nebula team on that. 
Screen 3: please remove the Ready to Apply? header when we have the SIP card showing on the page.
Screen 4: no changes
Screen 5: please remove the Ready to Apply? header when we have the SIP card showing on the page.

File: content/pages/burials-and-memorials/survivor-and-dependent-benefits.md [hunk 5]
```diff
@@ -29,3 +34,11 @@ relatedlinks:
 We give the surviving spouse, children, and parents of deceased Servicemembers and Veterans a range of benefits. These may include tax-free monetary benefits and help paying for the deceased's funeral service.
 
 </div>
+
+### Ready to apply?
```
Comment (jbalboni) [inline content/pages/burials-and-memorials/survivor-and-dependent-benefits.md]: Is the intentional? If it didn't have an apply link before, do we want it to have the same treatment as the main burials page, where we hid the apply button but show a saved application (if it exists)?

**Review by jbalboni — APPROVED** [review]
LGTM, had one question about one of the additions.

File: content/pages/burials-and-memorials/survivor-and-dependent-benefits.md (hunk 5 above)
Comment (cehsu) [reply content/pages/burials-and-memorials/survivor-and-dependent-benefits.md]: Good question, @peggygannon echoed your idea above, this was probably the intention of the designs in the original issue as well. This has been changed and the screenshot updated. Similar change will be made for pension and hca landing pages.

**Review by peggygannon — APPROVED** [review]

**Comment by peggygannon** [comment]
@cehsu LGTM! thanks.

**Pull request merged**